app/cafe.py

```diff
@@ -0,0 +1,26 @@
+import datetime
+from .errors import (
+    NotVaccinatedError, OutdatedVaccineError, NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> None:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError(f"{visitor['name']} is not vaccinated.")
+
+        expiration_date = visitor["vaccine"].get("expiration_date")
+        if expiration_date < datetime.date.today():
+            raise OutdatedVaccineError(
+                f"{visitor['name']}'s vaccine is outdated."
+            )
+
+        if not visitor.get("wearing_a_mask", False):
+            raise NotWearingMaskError(
+                f"{visitor['name']} is not wearing a mask."
+            )
+
+        return f"Welcome to {self.name}"
```

Comment:
The method should return a string as specified in the task description. Currently, it only returns a string when no exceptions are raised. Consider handling the exceptions in a way that ensures a string is returned in all cases.